internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -47,7 +47,7 @@ type templater interface {
 	Template() (string, error)
 }
 
-type svc struct {
+type wkld struct {
```

Comment:
What do you think about adding a comment here for what workload means? as I don't think it'll be as obvious to other readers as service.

```suggestion
// wkld represents a containerized workload running on Amazon ECS.
// A workload can be a long-running service, an ephemeral task, or a periodic task.
type wkld struct {
```